disperser/apiserver/server.go

```diff
@@ -778,6 +795,15 @@ func (s *DispersalServer) Start(ctx context.Context) error {
 	return nil
 }
 
+func (s *DispersalServer) LoadAllowlist() {
+	al, err := ReadAllowlistFromFile(s.rateConfig.AllowlistFile)
+	if err != nil {
+		s.logger.Error("failed to load allowlist", "err", err)
+		return
+	}
+	s.rateConfig.Allowlist = al
```

Comment:
Does this need any locks?

Reply:
I don't think this really needs to be synchronized. 
The worst case is that a potentially disallowed request goes through at the moment new allowlist is loaded (or vice versa)